lib/modal.js

```diff
@@ -0,0 +1,65 @@
+class Modal extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    this._render();
+  }
+
+  _render() {
+    this.id = "modal-overlay";
+
+    const modalContainer = document.createElement("div");
+    modalContainer.id = "modal-container";
+    console.log({ modalContainer });
```

Comment:
Remove these console.logs before shipping this code :hugs:

Reply:
Done!